component/app.jsonnet

```diff
@@ -2,9 +2,10 @@ local kap = import 'lib/kapitan.libjsonnet';
 local inv = kap.inventory();
 local params = inv.parameters.openshift4_operators;
 local argocd = import 'lib/argocd.libjsonnet';
+local instance = inv.parameters._instance;
 
-local app = argocd.App('openshift4-operators', params.namespace);
+local app = argocd.App(instance, params.namespace);
 
 {
-  'openshift4-operators': app,
+  [instance]: app,
 }
```

Comment:
This isn't strictly necessary, as long as the component author(s) ensure that the generated manifests for each instance use unique names (which is a valid way to handle multi-instance).

But since we've already talked about adding support to the component to manage more manifests than just the namespace and operatorgroup for each instance, the change probably makes sense.

However, this should be a separate PR so that the change is indicated in the component changelog.

Reply:
https://github.com/appuio/component-openshift4-operators/pull/23